examples/ethereum-tracker/src/contract.rs

```diff
@@ -113,7 +113,7 @@ impl EthereumTrackerContract {
             r#"query {{ readTransferEvents(endBlock: {end_block}) }}"#
         ));
 
-        let application_id = self.runtime.application_id();
+        let application_id = self.runtime.application();
```

Comment:
This PR ruins the field/variable naming convention in many places.

Reply:
See https://github.com/linera-io/linera-protocol/pull/3582 for alternative.